### DISCUSSION TITLE
Universal Declaration of Rights (Particularly Article 19) as Architecture Guidance

### DISCUSSION THREAD

**Discussion by bobwyman** [bluesky-social/atproto#106] [General Discussion]
I suggest that the [Universal Declaration of Rights](https://www.un.org/en/about-us/universal-declaration-of-human-rights#:~:text=Article%2019,media%20and%20regardless%20of%20frontiers.), particularly Article 19, should inform and provide a context for discussions of the architecture and its implementation. Article 19 reads:

> Everyone has the right to freedom of opinion and expression; this right includes freedom to hold opinions without interference and to seek, receive and impart information and ideas through any media and regardless of frontiers.

bob wyman

**Comment by arcalinea** [comment]
That's a good point -- I've thought about the framing of human rights as a way to describe the need for a distinction between "speech" and "reach". Will take this into consideration in future writing that focuses more on the high-level context for this architecture.
